lib/screens/action/action_flow.dart

```diff
@@ -0,0 +1,141 @@
+import 'package:provenance_dart/wallet_connect.dart';
+import 'package:provenance_wallet/common/flow_base.dart';
+import 'package:provenance_wallet/common/pw_design.dart';
+import 'package:provenance_wallet/common/widgets/modal/pw_modal_screen.dart';
+import 'package:provenance_wallet/screens/action/action_list/action_list_bloc.dart';
+import 'package:provenance_wallet/screens/action/action_list/action_list_screen.dart';
+import 'package:provenance_wallet/screens/transaction/transaction_confirm_screen.dart';
+import 'package:provenance_wallet/services/models/account.dart';
+import 'package:provenance_wallet/services/models/requests/send_request.dart';
+import 'package:provenance_wallet/services/models/requests/sign_request.dart';
+import 'package:provenance_wallet/services/models/wallet_connect_session_request_data.dart';
+import 'package:provenance_wallet/util/assets.dart';
+import 'package:provenance_wallet/util/get.dart';
+import 'package:provenance_wallet/util/messages/message_field_name.dart';
+import 'package:provenance_wallet/util/strings.dart';
+
+class ActionFlow extends FlowBase {
+  const ActionFlow({required this.account, Key? key}) : super(key: key);
+
+  final Account account;
+
+  @override
+  State<StatefulWidget> createState() => ActionFlowState();
+}
+
+class ActionFlowState extends FlowBaseState implements ActionListNavigator {
+  @override
+  void initState() {
+    super.initState();
+
+    get.pushNewScope(scopeName: "ActionScope");
```

Comment:
I thought that the standard we discussed was registering blocs and local services if we needed them and then "unregistering" them when done.

Reply:
In this case, I think what we are trying to achieve is having multiple `ActionListBloc`s exist at the same time because they can be for separate accounts, and are separate from the current selected account used on the dashboard. We could register them with a name, but then we need to pass the name into the components that use it, so maybe we might as well just pass the bloc itself instead...

Reply:
The idea was that we pop the scope when the workflow is disposed and all its children are cleaned up since they should not be needed anymore.

Reply:
nothing should use that actionList bloc except for the action List screen